.gitignore

```diff
@@ -14,7 +14,7 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
+#lib/
```

Comment:
Don't know where this .gitignore template file came from, I can't remember adding it but I suspect that we might have got it for free on creating an empty/new repo ? Happy to nuke the line rather than comment it out ... your call :smile: